src/components/QueryPanel/QueryPanel.tsx

```diff
@@ -26,8 +29,11 @@ export default function QueryPanel({
   setQuery,
   runQuery,
 }: QueryPanelProps) {
+  const {queryPanelWidth, setQueryPanelWidth} = useContext(
+    ExplorerPanelsContext
+  );
   return (
-    <div {...stylex.props(styles.main)}>
+    <div {...stylex.props(styles.main)} style={{width: queryPanelWidth || 360}}>
```

Comment:
These components can exist in contexts where their width is determined by an external factor, like in a VS Code Side Panel, so they should probably default to "auto" or "100%" not a fixed width.